server/build.gradle

```diff
@@ -151,8 +151,8 @@ application {
     mainClass.set("com.objectcomputing.checkins.Application")
 }
 java {
-    sourceCompatibility = JavaVersion.toVersion("17")
-    targetCompatibility = JavaVersion.toVersion("17")
+    sourceCompatibility = JavaVersion.toVersion("21")
```

Comment:
These settings are also in the web-ui/build.gradle and set to 17. Probably doesn't matter, but might be worth bumping them as well.

Reply:
Good catch. Fixed.